filip/clients/ngsi_v2/cb.py

```diff
@@ -523,7 +524,8 @@ def update_entity(self,
             raise
 
     def delete_entity(self, entity_id: str, entity_type: str,
-                      delete_devices: bool = False) -> None:
+                      delete_devices: bool = False,
+                      iota_url: AnyHttpUrl = settings.IOTA_URL) -> None:
```

Comment:
Well, you may also have entities that do not correlate to a device in the IoTA agent. Hence document it that it is required when delete_devices is True

Reply:
As the method does not fail if no correlating device(s) exists, is it necessary to make that annotation?
The current statement holds True in all case: Delete any device that corresponds to this entity.